src/docs.js

```diff
@@ -4,44 +4,60 @@ let assert = require('assert');
 let fs = require('mz/fs');
 let path = require('path');
 let recursiveReadSync = require('recursive-readdir-sync');
+let zlib = require('zlib');
 
 async function documenter(options) {
-
   options = _.defaults({}, options, {
+    metadata: null,
     schemas: [],
     docsFolder: null,
```

Comment:
This isn't part of this review, but while I notice it I may as well mention that we should have this default to a `docs/` directory rather than defaulting to null.